src/features/SearchMovie/ui/SearchBar.tsx

```diff
@@ -0,0 +1,47 @@
+import { ChangeEvent, FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Paths } from 'shared/types';
+import { useLocalStorage } from 'shared/lib/hooks';
+import './SearchBar.css';
+
+interface Props {
+  onClickCheck: (request: string) => void;
+}
+
+export const SearchBar: FC<Props> = ({ onClickCheck }) => {
+  const [request, setRequest] = useLocalStorage();
+
+  const onChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    const title = event.target.value;
+    setRequest(title);
+  };
+
+  const onClick = (): void => {
+    const title = request.trim();
+    onClickCheck(title);
+  };
+
+  return (
+    <>
+      <h1>Movies</h1>
+      <form className="searchBar">
+        <input
+          type="search"
+          value={request ? request : ''}
```

Comment:
```suggestion
          value={request || ''}
```
Логические операторы позволят сократь код